h/api/storage.py

```diff
@@ -33,15 +34,15 @@ def annotation_from_dict(data):
     return models.elastic.Annotation(data)
 
 
-def fetch_annotation(request, id, _postgres=None):
+def fetch_annotation(request, annotation_id, _postgres=None):
```

Comment:
Ok. Nobody _ever_ uses the `id` function. Let's not follow Pylint's advice if it means writing worse code. Does the `good-names` directive in `.prospector.yaml` not catch this? I'm 👍 on disabling that warning if you want to.